packages/block-editor/src/components/list-view/block.js

```diff
@@ -149,15 +134,6 @@ function ListViewBlock( {
 		{ 'is-visible': isHovered || isFirstSelectedBlock }
 	);
 
-	// If ListView has experimental features related to the Persistent List View,
```

Comment:
I do not think this is needed any longer. Focus is managed elsewhere.

Reply:
From working through the failing tests, it appears we still need this useEffect. Without it, when opening the list view, it doesn't redirect focus to the currently selected block. The test that fails is `'selects the default paragraph block after removing all blocks'`.

Reply:
Added back.